templates/homepage-welcome-band.tmpl

```diff
@@ -0,0 +1,15 @@
+<div class="grid-wrapper homepage-welcome-band content-slim"
+     id="welcome">
+  <div class="width-12-12 width-12-12-m">
+    <div class="welcome-row">
+      <div class="welcome-box-left">
+        <h2>{{ homepage.welcome.title }}</h2>
+        <p>{{ homepage.welcome.intro }}</p>
+      </div>
+      <div class="welcome-box-right max-width-prose">
+        <iframe width="420" height="300" src="{{ homepage.welcome.url }}">
```

Comment:
You could probably just add the closing tag here like this:

``<iframe width="420" height="300" src="{{ homepage.welcome.url }}" />``

Reply:
If you can implement that change for me I would appreciate it.  My focus is on other activities today.

Reply:
Nevermind. Change implemented.